AgentInterfaces/SessionHostsStartInfo.cs

```diff
@@ -142,4 +142,12 @@ public class LinuxInstrumentationConfiguration
         /// </summary>
         public bool IsEnabled { get; set; }
     }
+
+    public class WindowsInstrumentationConfiguration
```

Comment:
shouldn't an instance of `WindowsInstrumentationConfiguration`be included in the `SessionHostsStartInfo` class?